project.clj

```diff
@@ -13,21 +13,28 @@
   ;; requires lein 2.2.0+.
   :pedantic? :abort
 
+  :exclusions [org.clojure/clojure]
```

Comment:
Was this technically necessary or did it just seem like A Good Thing To Do.

TBH I'm not sure what the best approach is here.  I didn't realize that tk-status was doing this as well.  After some more thought it seems reasonable.